### DISCUSSION TITLE
Release notes does not work when using cherry picking

### DISCUSSION THREAD

**Discussion by germa89** [community/community#19067] [Actions]
Hi there!

It seems that if you do cherry picking in your release branch and then release, the "Generate release notes" does not work. It will just add the link to a diff between the release and the previous one.

**Comment by MartinX3** [comment]
I run in the same problem.
The auto generation lists only **pull requests**, not normal commits.

My workflow:
1. master & dev branch
2. commit stuff to the dev branch
3. squash the dev branch commits for a feature to a single commit
4. rebase master on the dev branch
5. tag the single commit
6. generate a release

I would need to open and rebase merge a new PR each time from my IDE.
(The releases are autogenerated on each git tag with a github-action) 

Also I see that our request is a duplicate of https://github.com/github-community/community/discussions/14517

**Comment by lucacome** [comment]
if I cherry-pick the commit of a PR to `main` in my release branch, I would expect the notes to add that PR when generating release notes that target that release branch.

**Comment by josienb** [comment]
I've asked a related question for the "Generate release notes" button in the GitHub UI here: https://github.com/orgs/community/discussions/113181. This issue does not have much traction, it seems.
